extensions/amp-byside-content/0.1/amp-byside-content.js

```diff
@@ -179,32 +227,45 @@ export class AmpBysideContent extends AMP.BaseElement {
 
     this.element.appendChild(this.getOverflowElement_());
     this.applyFillContent(iframe);
+    this.propagateAttributes(ATTRIBUTES_TO_PROPAGATE, iframe);
 
     return this.composeSrcUrl_().then(src => {
       this.iframeSrc_ = assertHttpsUrl(src, this.element, this.getName_());
       iframe.src = this.iframeSrc_;
 
-	  const unlisten = listenFor(iframe, 'embed-size', this.boundUpdateSize_);
-	  this.unlisteners_.push(unlisten);
-
-	  this.element.appendChild(iframe);
-      return (this.iframePromise_ = this.loadPromise(iframe));
+      const unlisten = listenFor(iframe, 'embed-size', this.boundUpdateSize_);
+      this.unlisteners_.push(unlisten);
+
+      this.container_.appendChild(iframe);
+
+      return (this.iframePromise_ = this.loadPromise(iframe).then(() => {
+        // On iOS the iframe at times fails to render inside the `overflow:auto`
+        // container. To avoid this problem, we set the `overflow:auto` property
+        // 1s later via `amp-active` class.
+        if (this.container_ != this.element) {
```

Comment:
we have been dealing with similar problems for scrollable areas in iOS. The 1000 timeout is arbitrary though. Maybe you wanna run this first time embed-size message is received.

Reply:
Applied the same behaviour to when the embed-size message is received.